cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/selector/BaseStorageSelector.java

```diff
@@ -0,0 +1,10 @@
+package com.linkedin.openhouse.cluster.storage.selector;
+
+/** An abstract class for all Storage selectors */
+public abstract class BaseStorageSelector implements StorageSelector {
+
+  @Override
+  public String getName() {
+    return this.getClass().getSimpleName();
```

Comment:
does this return `BaseStorageSelector` or `DefaultStorageSelector`?

Reply:
It always returns the implementing class's getName method. In this case DefaultStorageSelector